Doc/reference/compound_stmts.rst

```diff
@@ -820,7 +820,9 @@ and binds no name.  Syntax:
 .. productionlist:: python-grammar
    wildcard_pattern: '_'
 
-``_`` is a :ref:`soft keyword <soft-keywords>`.
+``_`` is a :ref:`soft keyword <soft-keywords>` within any pattern,
+but only within patterns.  It is an identifier, as usual, even within
+``match`` headers, ``guards``, and ``case blocks``.
```

Comment:
```suggestion
``match`` headers, ``guards``, and ``case`` blocks.
```
"blocks" is not a piece of code, so it should not be in monospace font.

Reply:
Correct.  In the doc, I see `case_block` (grammar term) and "``case`` block"